pandas/core/generic.py

```diff
@@ -1084,6 +1084,9 @@ def to_hdf(self, path_or_buf, key, **kwargs):
                        / selecting subsets of the data
         append   : boolean, default False
             For Table formats, append the input data to the existing
+        data_columns : list of columns to create as data columns, or True to
+            use all columns. This will create additional indexed columns for
+            on-disk queries, by default only 'index' and 'columns' are indexed.
```

Comment:
The same `columns` as written in your docs? See here: http://pandas.pydata.org/pandas-docs/stable/io.html#querying-a-table